tests/shell_lock/tests/01-run.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2021 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+import time
+
+from testrunner import run
+
+PASSWORD_CORRECT = "password"
+PASSWORDS_INCORRECT = [
+    "pass",
+    "word",
+    "asswor",
+    "passw0rd",
+    "password_",
+    "_password"
+]
+
+EXPECTED_HELP = (
+    'Command              Description',
+    '---------------------------------------',
+    'reboot               Reboot the node',
+    'version              Prints current RIOT_VERSION',
+    'pm                   interact with layered PM subsystem',
+    'lock                 Lock the shell'
+)
```

Comment:
tl;dr:
This test will now fail due to changed order of shell commands.

With `SHELL_COMMAND()` now being used to register shell commands, the shell commands are now alphabetically sorted (except for the commands supplied as parameter to `shell_run()`). Note that `SHELL_COMMAND()` can also be used just fine from applications, making use supplied shell commands first class citizens.